en/docs/learn/configuring-outbound-provisioning.md

```diff
@@ -0,0 +1,180 @@
+
+    | **Identity Provider Claim URI**       | **Value**               |
+    |------------------------------------|---------------------|
+    | Alias                              | User                |
+    | Email                              | user@wso2.com       |
+    | EmailEncodingKey                   | UTF-8               |
+    | LanguageLocaleKey                  | en_US               |
+    | LastName                           | Jackson             |
+    | LocaleSidKey                       | en_US               |
+    | ProfileId                          | 2F00e6F000003nIYI   |
+    | TimeZoneSidKey                     | America/Los_Angeles |
+    | UserPermissionsCallCenterAutoLogin | false               |
+    | UserPermissionsMarketingUser       | false               |
+    | UserPermissionsOfflineUser         | false               |
+    | Username                           | user@wso2.com       |
+
+10. Expand **Outbound Provisioning Connectors > Salesforce Provisioning Connector**. Fill the relevant fields. 
+    1. Select **Enable Connector** to enable Salesforce provisioning.
+    2. Enter the **API version**. This refers to the Salesforce API version you are using. Follow the steps below to get the API version.
+       
+        ??? note "Get API version: Click to expand"
+            1. Log in to [Salesforce](https://login.salesforce.com).
+            2. Search for "API" on the Quick Find search box and click **API** on the search results. 
+            3. Generate any of the WSDLs to see the API version. On the top of the WSDL you will see text similar to the following from which you can get the Salesforce API version. 
+            ``` xml
+            Salesforce.com Enterprise Web Services API Version 45.0
+            ```
+
+            4. Enter this value as the **API Version** in WSO2 Identity Server in the following format: "V<version number>".
+               <br/>For example `v45.0`.
+
+    3. Create a Salesforce domain if you do not already have one and enter the **Domain Name**. Follow the steps given below to create one. Make sure that you add the `https://` prefix to the **Domain Name** as shown in the example given below. 
+
+        ```xml
+        https://wso2-dev-ed.my.salesforce.com
+        ```
+
+        ??? note "Create Salesforce domain: Click to expand"
+            1. Log in to [Salesforce](https://login.salesforce.com).
+            2. Search for "My Domain" on the Quick Find search box found on the left navigation panel and click **My Domain**.
+            3. Enter a name for your domain. You can check if the domain name is available by clicking **Check Availability**. 
+            4. Click **Register** to register your new domain. Your domain name will be similar to the following: `wso2-dev-ed.my.salesforce.com`.
+            5. Once the domain is registered, Salesforce will take 2 minutes to test it. Click **Click here to Login** to test it out. 
```

Comment:
Shall we rephrase this? 
Not clear if Salesforce takes 2 minutes to test it automatically or if have to click on **Click here to Login**.